src/app/app.module.ts

```diff
@@ -23,9 +25,9 @@ import { RecipeEffects } from './recipes/store/recipe.effect';
     AppRoutingModule,
     HttpClientModule,
     StoreModule.forRoot(fromApp.appReducer),
-    EffectsModule.forRoot([AuthEffects, RecipeEffects]),
+    EffectsModule.forRoot([AuthEffects, RecipeEffects, ShopingListEffects, ProfileEffects]),
```

Comment:
Propose you to add each array element from new line (better from git side, you will add new line, not modify previous line) like
`EffectsModule.forRoot([`
`    AuthEffects,`
`    RecipeEffects,`
`    ShopingListEffects,`
`    ProfileEffects,`
`])`
It isn't problem now, but it's a good practice